src/ekat/logging/ekat_log_mpi.hpp

```diff
@@ -0,0 +1,66 @@
+#ifndef EKAT_LOG_MPI_HPP
+#define EKAT_LOG_MPI_HPP
+
+#include "ekat/mpi/ekat_comm.hpp"
+#include "spdlog/sinks/null_sink.h"
+#include <mpi.h>
+
+// This file contains policy classes for choosing the behavior of the ekat
+// logger with respect to mpi ranks.
+// Each policy must implement the
+//
+// static void update_sinks(std::vector<spdlog::sink_ptr>& sinks)
+// static std::string name_append_rank(const std::string& name)
```

Comment:
Maybe consider `name_with_rank`, since the function returns the logger's name with the rank appended.

Reply:
Or possible `append_rank`, if you want to be "verby" in the context of this policy.